lib/figures/bishop.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require_relative 'figure'
+
+class Bishop
+  attr_reader :row, :column, :possible_moves, :color, :display
+
+  include Figure
+
+  def initialize(color, row, column)
+    @color = color
+    @row = row
+    @column = column
+    @display = { white: '♝', black: '♗' }
```

Comment:
Why keep both white and black in here? We know what color the instance is, right?

In instance variables, you are interested in only keeping the `variables`, that is, things that _vary_, things that represent that specific object.

Your `@display` variable does not represent your entity, it's a configuration variable. Either pick which one is correct and only use that, or turn that into a method. Same goes for `@move_pattern` method.